Source/Core/VideoCommon/VideoBackendBase.cpp

```diff
@@ -200,61 +198,58 @@ u16 VideoBackendBase::Video_GetBoundingBox(int index)
   return result;
 }
 
-// This function is called at static initialization, so we can't rely on s_default_backend being set
+static VideoBackendBase* GetDefaultVideoBackend()
+{
+  const auto& backends = VideoBackendBase::GetAvailableBackends();
+  if (backends.empty())
+    return nullptr;
+  return backends.front().get();
+}
+
 std::string VideoBackendBase::GetDefaultBackendName()
 {
-#ifdef HAS_OPENGL
-  return OGL::VideoBackend::NAME;
-#elif defined(_WIN32)
-  return DX11::VideoBackend::NAME;
-#else
-  return Vulkan::VideoBackend::NAME;
-#endif
+  return GetDefaultVideoBackend()->GetName();
```

Comment:
Is this a good idea when `GetDefaultVideoBackend` can _theoretically_ return `nullptr`?

Reply:
Whoops, good catch